sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -311,6 +376,108 @@ def __repr__(self):
         )
 
 
+class HealthcareEntity(DictMixin):
+    """HealthcareEntity contains information about a Healthcare entity found in text.
+
+        :ivar str text: Entity text as appears in the request.
+        :ivar str category: Entity category, such as Dosage or MedicationName, etc.
+        :ivar str subcategory: Entity subcategory.  # TODO: add subcategory examples
+        :ivar int offset: The Healthcare entity text offset from the start of the document.
+        :ivar int length: The length of the Healthcare entity text.
+        :ivar float confidence_score: Confidence score between 0 and 1 of the extracted
+            entity.
+        :ivar links: A collection of entity references in known data sources.
+        vartype links: list[~azure.ai.textanalytics.HealthcareEntityLink]
+    """
+
+    def __init__(self, **kwargs):
+        self.text = kwargs.get("text", None)
+        self.category = kwargs.get("category", None)
+        self.subcategory = kwargs.get("subcategory", None)
+        self.offset = kwargs.get("offset", None)
+        self.length = kwargs.get("length", None)
+        self.confidence_score = kwargs.get("confidence_score", None)
+        self.links = kwargs.get("links", [])
+
+    @classmethod
+    def _from_generated(cls, healthcare_entity):
+        return cls(
+            text=healthcare_entity.text,
+            category=healthcare_entity.category,
+            subcategory=healthcare_entity.subcategory,
+            offset=healthcare_entity.offset,
+            length=healthcare_entity.length,
+            confidence_score=healthcare_entity.confidence_score,
+            links=[HealthcareEntityLink(id=l.id, data_source=l.data_source) for l in healthcare_entity.links] if healthcare_entity.links else None
+        )
+
+    def __repr__(self):
+        return "HealthcareEntity(text={}, category={}, subcategory={}, offset={}, length={}, confidence_score={},\
+        links={})".format(
+            self.text,
+            self.category,
+            self.subcategory,
+            self.offset,
+            self.length,
+            self.confidence_score,
+            repr(self.links)
+        )[:1024]
+
+
+class HealthcareRelation(DictMixin):
+    """
+    HealthcareRelation contains information describing a relationship between two entities found in text.
+
+    :ivar str type: The type of relation, such as DosageOfMedication or FrequencyOfMedication, etc.
+    :ivar bool is_bidirectional: Boolean value indicating that the relationship between the two entities is
+        bidirectional.  If true the relation between the entities is bidirectional, otherwise directionality
+        is source to target.
+    ivar source: A reference to an extracted Healthcare entity representing the source of the relation.
```

Comment:
couple missing `:` prefixes here too. every docstring description needs to start with `:`

Reply:
See above comment.